ee/ucp/admin/configure/join-nodes/join-linux-nodes-to-cluster.md

```diff
@@ -99,7 +99,7 @@ To promote or demote a manager node:
 4.  Click **Save** and wait until the operation completes.
 5.  Navigate to the **Nodes** page, and confirm that the node role has changed.
 
-If you're load-balancing user requests to Docker Enterprise across multiple manager
+If you are load-balancing user requests to Docker Enterprise across multiple manager
 nodes, don't forget to remove these nodes from your load-balancing pool when
```

Comment:
Edit suggestion: 

Replace the second clause of the sentence with: "remember to remove these nodes from your load-balancing pool when demoting them to workers."